sys/include/ubjson.h

```diff
@@ -17,9 +17,9 @@
  */
 
 /**
- * @defgroup    sys_ubjson Universal Binary JSON
- * @ingroup     sys
- * @brief       A library to read and write UBJSON serialized data.
+ * @defgroup sys_ubjson  Universal Binary JSON deserializer
+ * @ingroup  sys_serialization
+ * @brief    Provides a UBJSON deserializer library to RIOT
```

Comment:
Still weird, that the original doc said, the lib is able to read and write UBJSON, but your new doc states its only able to deserialize (i.e. read)...

Reply:
You are right. Sorry